Source/Core/Core/ConfigManager.cpp

```diff
@@ -325,7 +325,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
 
   network->Set("SSLDumpRead", m_SSLDumpRead);
   network->Set("SSLDumpWrite", m_SSLDumpWrite);
-  network->Set("SSLVerifyCert", m_SSLVerifyCert);
+  network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
```

Comment:
To keep it consistent, shouldn't the other options be renamed as well? ```SSLDumpRootCertificateAuthorities``` and ```SSLDumpPeerCertificates```? Same concerns for the member variables.